webapp/channels/src/components/gif_picker/components/App/index.tsx

```diff
@@ -7,14 +7,13 @@ import {connect} from 'react-redux';
 import {saveAppProps} from 'mattermost-redux/actions/gifs';
 
 import Header from 'components/gif_picker/components/Header';
-import {appProps} from 'components/gif_picker/gif_picker';
 
 const mapDispatchToProps = ({
     saveAppProps,
 });
 
 type Props = {
-    appProps: typeof appProps;
+    appProps: any;
```

Comment:
Any chance we can have this not as an any?

Reply:
so explanation for this is after this PR we will be removing all removing all the old code of gif picker these files included. the reason it is not removed in this pr is because I didn't wanted to increase the diff